requirements.txt

```diff
@@ -1,2 +1,2 @@
 argparse; python_version < '2.7'
-# ovirt-engine-sdk >= 3.6  # It is not available in pip repo
+ovirt-engine-sdk-python == 3.6.9.2
```

Comment:
I think one of these constrains (== 3.6.9.2 or < 4) should remain there.
I know that python module name is different:
* ovirtsdk
* ovirtsdk4

and rpm distribution as well:
* ovirt-engine-sdk-python
* python-ovirt-engine-sdk4

but unfortunately on PyPi site both version are distributed under same package:
* ovirt-engine-sdk-python

https://pypi.python.org/pypi/ovirt-engine-sdk-python/4.1.3
https://pypi.python.org/pypi/ovirt-engine-sdk-python/3.6.9.2

In addition this fact causes difficulties to have both versions installed from PyPi at same time.